scripts/deploy/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "cht-deploy",
+  "version": "1.0.0",
+  "description": "## Deploy the CHT",
+  "main": "cht-deploy",
```

Comment:
minor, I think [main](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#main) should be src/install.js or something similar as cht-deploy is not intended to be imported

Reply:
handled.